setup.cfg

```diff
@@ -35,6 +35,7 @@ install_requires =
     build
     click >= 8
     pip >= 22.2
+    pep517
```

Comment:
This is being replaced by `pyproject-hooks` across the ecosystem now. We should do the same.

Reply:
@apljungquist did you miss this?

Reply:
Oops, I pushed the wrong commit last weekend.